NuRadioReco/detector/generic_detector.py

```diff
@@ -117,14 +138,77 @@ def add_generic_station(self, station_dict):
         if station_dict['station_id'] in self._buffered_stations.keys():
             logger.warning('Station with ID {} already exists in buffer. Cannot add station with same ID'.format(station_dict['station_id']))
             return
+
         for key in self.__default_station.keys():
             if key not in station_dict.keys():
                 station_dict[key] = self.__default_station[key]
         self._buffered_stations[station_dict['station_id']] = station_dict
+
         if self.__default_station_id not in self._buffered_channels.keys():
             self._buffer(self.__default_station_id)
         self._buffered_channels[station_dict['station_id']] = {}
+
         for i_channel, channel in self._buffered_channels[self.__default_station_id].items():
             new_channel = copy.copy(channel)
             new_channel['station_id'] = station_dict['station_id']
             self._buffered_channels[station_dict['station_id']][channel['channel_id']] = new_channel
+
+    def add_station_properties_for_event(self, properties, station_id, run_number, event_id):
```

Comment:
from the function definition it is not clear what 'properties' is. Can you add a standard doc string with more documentation? Is 'properties' a dictionary with the detector description parameters?

Reply:
Done